### DISCUSSION TITLE
Abandoned devices - how to delete?

### DISCUSSION THREAD

**Discussion by AminShAT** [SukramJ/aiohomematic#1581] [Q&A] [answered]
Hi, I have a couple of devices, that I replaced in Homematic. The devices do not appear anymore in home assistant, however the entities still do (with the state unavailable). When I click on the gear icon in order to delete them, it says, they do not have a unique ID and the can not be deleted. I tried the service recorder:purge entities as well, did not work either.

how can I get rid of these entities in Homeassistant?

**Comment by SukramJ** [comment, marked as answer]
Try to delete the statistics (dev tools -> statistics) first.
